### PR TITLE
Improving output of Cucumber tests and re-organizing feature files 

### PR DESCRIPTION
**Pull request by hferentschik**
Addresses issue #419 and #425

### CODE REVIEW

File: features/docker.feature [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I run `bundle exec vagrant up --provider <provider>`
+    Then the exit status should be 0
```
Comment (brgnepal) [inline features/docker.feature:25]: We can include this too here to verify message 
https://github.com/projectatomic/vagrant-service-manager/blob/master/features/adb-openshift.feature#L29-L34

File: features/docker.feature [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+          kubernetes  display information and environment variables for kubernetes
+
+    If OBJECT is omitted, display the information for all active services
+
+    Options:
+          --script-readable  display information in a script readable format.
+          -h, --help         print this help
+    """
+
+    ####################################################################################################################
```
Comment (brgnepal) [inline features/docker.feature]: This comment for ENV should start at line 73

File: features/docker.feature [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+          -h, --help         print this help
+
+    Examples:
+          vagrant service-manager box version
+          vagrant service-manager box version --script-readable
+          vagrant service-manager box ip
+          vagrant service-manager box ip --script-readable
+    """
+
+    When I successfully run `bundle exec vagrant service-manager box ip`
```
Comment (brgnepal) [inline features/docker.feature:70]: I think you left to test `box version` here

**Comment by hferentschik** [comment]
@LalatenduMohanty, @budhrg - new pull request. Commits are split now across two issues as discussed. Commit 692c is still separate, since it technically is not a cleanup, but I rather disable a test due to the OpenShift issue.

File: features/docker.feature (hunk 1 above)
Comment (hferentschik) [reply features/docker.feature:25]: You mean

```
==> default: Docker service configured successfully...
```

+1

File: features/docker.feature (hunk 3 above)
Comment (hferentschik) [reply features/docker.feature:70]: AFAICT 'box version' was not tested in box.feature either. I just combined the existing tests. I did not add new ones.

File: features/docker.feature (hunk 1 above)
Comment (brgnepal) [reply features/docker.feature:25]: @hferentschik Yes

File: features/docker.feature (hunk 2 above)
Comment (hferentschik) [reply features/docker.feature]: Ahh, thanks.

File: features/docker.feature (hunk 3 above)
Comment (brgnepal) [reply features/docker.feature:70]: @hferentschik Looks like we missed. As you know we added tests after few initial features were already existing, probably `box version` is one of them.

Would u like to add it here or in another PR ? For me, either way will work.

**Comment by hferentschik** [comment]
@budhrg pushed an update

File: features/docker.feature (hunk 3 above)
Comment (hferentschik) [reply features/docker.feature:70]: Let's do this in another pull request.

**Comment by LalatenduMohanty** [comment]
@hferentschik @budhrg LGTM.

**Comment by brgnepal** [comment]
@hferentschik Let me know if you are adding `systemctl start openshift` changes here and CI is passing, Locally it is passing but not CI for me. details here https://github.com/projectatomic/vagrant-service-manager/issues/415#issuecomment-256048733

**Comment by hferentschik** [comment]
> Let me know if you are adding systemctl start openshift changes here and CI is passing, Locally it is passing but not CI 

Hmm, I would have thought that would be a new pull request for issue #415

**Comment by brgnepal** [comment]
> Hmm, I would have thought that would be a new pull request for issue #415

Cool. Will merge this now.

**Comment by brgnepal** [comment]
Merged and thanks.

**Pull request closed without merging**